djangobench/utils.py

```diff
@@ -1,9 +1,6 @@
 import argparse
 import inspect
 import os
-
-# timeit uses either time.time() or time.clock() depending on which is more
-# accurate on the current platform:
```

Comment:
The comment didn't look useful to me and I didn't care to fight with isort.

Travis would be useful, but I don't have much expertise so it might take me a little while. I'll create a ticket instead.